demo_project/demo/views.py

```diff
@@ -264,7 +264,11 @@ def get_context_data(self, **kwargs):
         data_source = context["data_source"]
         point_chart = self.renderer.PointChart(data_source,
                                   options={'title': "Sales Growth"})
-        context["point_chart"] = point_chart
+        data_source = SimpleDataSource(data)
```

Comment:
@ashokjallepalli  I guess data_source is already defined in this method, do we need to define it again?